### ISSUE TITLE
[49.1+390] Missing unit groups in the lobby on 1st session load

### ISSUE THREAD

**Issue by Tecmes** [MegaMek/mekhq#2614]
I've noticed that every time I start a MM game from MHQ/AtB, if it's the first game I launch after starting MHQ fresh, there is always a whole unit group missing. It appears in the players list, but there is no units. It's like loading MM for the first time creates an issue (but heck I don't know).
Closing and restarting the game exactly the same way works the second time over.
I,ve set delay to 2000ms.

[megameklog.txt](https://github.com/MegaMek/mekhq/files/6479616/megameklog.txt)

**Comment by Tecmes** [comment]
Further testing: the issue happens *every time* on first load, and even sometimes on 2nd or 3rd load for especially busy lobbies.
When a group is missing, it's always the 2nd or 3rd group, i.e. the 1st or 2nd after your own units.

**Comment by Tecmes** [comment]
Well, not every time. At least in build 398 it doesn't happen with the new force I created.

**Comment by Tecmes** [comment]
IT seems it really is a display bug. The forces are here, but do not display in the lobby. Sometimes if you move the cursor on the area where they are supposed to be in the lobby, parts of it appear. And if you start the game, they are there.
I obsreve that the display tends to re-arrange itself after loading. Maybe that's when the issue pops up?

**Comment by Windchild292** [comment]
This needs retesting in the latest version to ensure it hasn't been fixed.

**Comment by Tecmes** [comment]
Yes, I've had it with 49.1. It's not very clear when, except that's when there's lots of units.

**Comment by IllianiCBT** [comment]
The last update to this issue was 4 years ago. As we've not seen any other reports of the behavior and as players tend to be using very large campaigns nowadays I'm going to close this case under the presumption that it's been fixed.
